fineract-provider/dev-dependencies.gradle

```diff
@@ -105,11 +105,12 @@ dependencies {
             'org.apache.activemq:activemq-broker',
             'javax.validation:validation-api',
     )
+    compile 'io.rest-assured:rest-assured'
```

Comment:
as above, I would suggest to remove this (and only `testCompile` it)

```suggestion
```

Reply:
Ok I will remove it.